certificate/templates/python_workshop_download.html

```diff
@@ -28,6 +28,26 @@ <h4 class="modal-title" id="invalidModalLabel">Invalid User/Email</h4>
                 </div>
               </div>
           </div>
+          <div class="modal fade" id="failModal" tabindex="-1" role="dialog" aria-labelledby="failModalLabel" aria-hidden="true">
+              <div class="modal-dialog">
+                <div class="modal-content">
+                  <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal">&times;</button>
+                    <h4 class="modal-title" id="invalidModalLabel">Certificate not Awarded</h4>
+                  </div>
+                  <div class="modal-body">
+                      {% if failed == 1 %}
+                        As per our records, you have not secured sufficient marks to get a certificate. Please try our <a href="https://python-workshops.fossee.in/self_workshop">self-learning course</a> to improve your knowledge of Python.
+                      {% else %}
```

Comment:
When is this else condition executed?
As from views failed is only passed as 1!